### PR TITLE
chore(sdk/go): rename everything from sdk to sdk/go

### PR DESCRIPTION
**Pull request by GeorgeMac**
After some discussion and exploration we decided to name the new SDK `go.flipt.io/flipt/sdk/go` for convenience.
We may consider moving to something terser in the future (e.g. to `go.flipt.io/sdk`).
However, that requires some more automation to continue developing it in this repo as a submodule.

This PR changes the references in this repo to the sub-directory path `sdk/go`.
It also sets the version of `rpc/flipt` used in `sdk/go/go.mod` to the new tagged version `v1.19.2`.

The TL;DR on the work needed to support a shorter import URL is:
The `go-import` meta tag for vanity URLs does not support sub-directories in git repositories.
This means you're forced to either using the directory as a suffix on your root vanity URL, or you have to move your directory out into the root of its own git project.
We would like to continue developing SDKs (potentially of various languages) in this repository under `sdk` directory.

### CODE REVIEW

**Comment by GeorgeMac** [comment]
This is currently failing because I have moved to Go 1.20 and and I synced all the go mods in the file.

There is a PR up to move Flipt to 1.20 where we can discuss it: https://github.com/flipt-io/flipt/pull/1427

File: internal/cmd/protoc-gen-go-flipt-sdk/go.mod [hunk 1]
```diff
@@ -2,6 +2,16 @@ module go.flipt.io/protoc-gen-go-flipt-sdk
 
 go 1.19
```
Comment (markphelps) [inline internal/cmd/protoc-gen-go-flipt-sdk/go.mod:3]: does this need to go to `1.20` as well?

**Review by markphelps — COMMENTED** [review]
2 minor comments, otherwise lgtm!

File: sdk/go/go.mod [hunk 2]
```diff
@@ -1,21 +1,21 @@
-module go.flipt.io/flipt/sdk
+module go.flipt.io/flipt/sdk/go
 
 go 1.18
```
[the remark this answers is not in the document]
Comment (GeorgeMac) [reply sdk/go/go.mod]: Same as above 👍

File: internal/cmd/protoc-gen-go-flipt-sdk/go.mod (hunk 1 above)
Comment (GeorgeMac) [reply internal/cmd/protoc-gen-go-flipt-sdk/go.mod:3]: or maybe not...

Comment (GeorgeMac) [reply internal/cmd/protoc-gen-go-flipt-sdk/go.mod]: Done, thanks for noticing!

**Review by markphelps — APPROVED** [review]
